other/MAINTAINING.md

```diff
@@ -41,8 +41,8 @@ as you want/need to. Nobody can ask any more of you than that.
 As a maintainer, you're fine to make your branches on the main repo or on your own fork. Either
 way is fine.
 
-When we receive a pull request, a travis build is kicked off automatically (see the `.travis.yml`
-for what runs in the travis build). We avoid merging anything that breaks the travis build.
+When we receive a pull request, a GitHub Actions build is kicked off automatically (see the `.github/`
```

Comment:
We use CircleCI, not sure why this was Travis and GH Actions are there.

Reply:
Alright, I fixed that